internal/controller/pkg/revision/establisher.go

```diff
@@ -394,6 +395,29 @@ func (e *APIEstablisher) enrichControlledResource(res runtime.Object, webhookTLS
 			conf.Spec.Conversion.Webhook.ClientConfig.Service.Namespace = e.namespace
 			conf.Spec.Conversion.Webhook.ClientConfig.Service.Port = ptr.To[int32](ServicePort)
 		}
+	case *v2alpha1.ManagedResourceDefinition:
```

Comment:
is this basically the exact same block we use for CRDs above? ^

Reply:
it is. I copy/pasted, I tried to `case A,B:` it but go did not like this because we are accessing member fields.